pycoal/mineral.py

```diff
@@ -2,132 +2,219 @@
 
 import spectral
 import pickle
+import numpy
 
 # define number of layers in normalized pixel
 NUMBER_OF_LAYERS = 128
 
-# define wavelength range
-WAVELENGTH_MIN = ???
-WAVELENGTH_MAX = ???
+# define wavelength range in micrometers
+WAVELENGTH_MIN = 0.38315
+WAVELENGTH_MAX = 2.5082
+
+class MineralClassification:
+    def __init__(self):
+        # maybe we could put the constants above in here?
+        # any other variables to move in here?
```

Comment:
@lewismc @heidiaclayton negative on that. `NUMBER_OF_LAYERS`, `WAVELENGTH_MIN`, and `WAVELENGTH_MAX` should only be used by the mineral classification. the output will be a classified file with a single layer.